build_test.go

```diff
@@ -570,6 +600,51 @@ func testBuild(t *testing.T, when spec.G, it spec.S) {
 					h.AssertNil(t, os.Remove(buildpackTgz))
 				})
 
+				when("buildpack is an older api version", func() {
```

Comment:
Given that we cover this in the builder package tests I don't see the point in duplicating this test coverage here, there  will just be more tests to maintain.